code/chart.js

```diff
@@ -1,4 +1,33 @@
-//DOM-selector for the canvas 👇
-const ctx = document.getElementById('chart').getContext('2d')
+const ctx = document.getElementById('chart').getContext('2d') // don't really get what this does
+
+const renderChart = (completedProjects) => {
+
+    const labels = [
+        'Projects done',
+        'Projects left to do'
+      ];
+    
+      const data = {
+        labels: labels,
+        datasets: [{
+          backgroundColor: 'rgb(217, 169, 165)',
+          label: 'Technigo Projects',
+          borderColor: 'rgb(192, 113, 106)',
+          data: [completedProjects, 19-completedProjects],
+        }]
+      };
+    
+      const config = {
+        type: 'bar',
+        data: data,
+        options: {
+            responsive: true,
```

Comment:
Did you want the chart to cover the page from side to side? Because the size of the chart is good for mobile size but displays quite big in desktop size.

Reply:
Thanks for noticing Ida! I thought about that too and want to change it for desktop. Do you know which element to add in that case?

Reply:
I found the instructions here: [https://www.chartjs.org/docs/latest/configuration/responsive.html](url)

Reply:
Oh, thanks! I got a 404 though :(